<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>46</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>46</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>261</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>주소 혹은 충전소 명을 입력하세요</string>
    </property>
   </widget>
   <widget class="QWebView" name="webView" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>441</width>
      <height>521</height>
     </rect>
    </property>
    <property name="url" stdset="0">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QPushButton" name="SendEmail">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Email전송</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>search</tabstop>
  <tabstop>comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search</sender>
   <signal>clicked()</signal>
   <receiver>comboBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SendEmail</sender>
   <signal>clicked()</signal>
   <receiver>comboBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
